src/utils/utils.ts

```diff
@@ -0,0 +1,4 @@
+import { randomBytes } from 'crypto';
+
+
+export const apiKey = randomBytes(64).toString('base64');
```

Comment:
Can't it be a function? To allow generating more than one apiKey in the future, then save in some kind of cache